tests/printer/expr/expected/binary.res.txt

```diff
@@ -472,22 +472,18 @@ foo :=
   | _ => 1
   }
 
-<div>
-  {possibleGradeValues
-  |> List.filter(g => g < state.maxGrade)
-  |> List.map(possibleGradeValue =>
+<div> {possibleGradeValues |> List.filter(g =>
```

Comment:
This code seems to be printed in a pretty hard to read way.
How hard is it to preserve the previous format?

Reply:
No idea! But I have a different approach in mind that should preserve pretty much all the formatting.

Reply:
The fact that it's unclear why the examples have changed, is more worrying than the changes themselves.